tests/test_cli.py

```diff
@@ -87,6 +87,38 @@ def test_init(app, template_entrypoints):
 
     assert current_search_client.indices.exists(list(search.mappings.keys()))
 
+    # Test option --index-name
```

Comment:
Great to test this!
A minor comment: does it make sense to maybe split in smaller tests?